test_accounts/appealable_issues_test_accounts.md

```diff
@@ -0,0 +1,27 @@
+# Sandbox Test Accounts for Appealable Issues API (v0)
+
+This document provides test account information for the [Appealable Issues API (v0)](https://developer.va.gov/explore/api/appealable-issues/docs?version=current). The API returns synthetic VA records for these test accounts in the sandbox environment.
+
+## Authentication and Authorization
+
+The authentication model for the [Appealable Issues API (v0)](https://developer.va.gov/explore/api/appealable-issues/docs?version=current) uses OAuth 2.0/OpenID Connect. The following authorization models are supported:
+- [Authorization code flow](https://developer.va.gov/explore/api/appealable-issues/authorization-code)
+- [Client Credentials Grant (CCG)](https://developer.va.gov/explore/api/appealable-issues/client-credentials)
+
+Login information for all test accounts can be found on the [developer portal](link-will-go-here-once-page-created).
+
+## Veteran Test Accounts
+
+These accounts can be used to test various data and response scenarios for the `GET /appealable-issues/{decisionReviewType}` endpoint. 
```

Comment:
Should we mention that they can pass any `decisionReviewType` and the same results will be provided regardless?